android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -124,6 +123,9 @@ class PolarPlugin :
         call: MethodCall,
         result: Result,
     ) {
+        if(call.method != "init" && wrapperInternal == null) {
```

Comment:
Why can't we automatically initialize the SDK once without having to call an init method?

Reply:
Yeh good point, hadn't thought about that